src/logger.js

```diff
@@ -225,9 +208,6 @@ function logToConsole(type, sender, msgShort, _extras) {
     } catch (e) {
       logLocal('error', 'logger', 'trying to log something illegal? msgShort: ' + msgShort + ' msgLong: ' + msgLong + ' extras: ' + extras + ' orig msg: ' + e, false, { extras: extras, error: e })
     }
-    if (type === 'CRITICAL') {
```

Comment:
just to point out... this is the place where we could do something like `if type CRITICAL or ERROR send to sentry` and then delete it from the backend

Reply:
agree but in the current implementation of the logger it will add just problems